ddtrace/appsec/iast/_taint_utils.py

```diff
@@ -353,6 +346,28 @@ def values(self):
         for _, v in self.items():
             yield v
 
+    # Django Query Dict support
+    def getlist(self, key, default=None):
+        return self._taint(self._obj.getlist(key, default=default), key)
+
+    def setlist(self, key, list_):
+        self._obj.setlist(key, list_)
+
+    def appendlist(self, key, item):
+        self._obj.appendlist(key, item)
+
+    def setlistdefault(self, key, default_list=None):
+        return self._taint(self._obj.setlistdefault(key, default_list=default_list), key)
+
+    def lists(self):
+        return self._taint(self._obj.lists(), self._origin_value)
+
+    def dict(self):
+        return self
+
+    def urlencode(self, safe=None):
+        return self._taint(self._obj.urlencode(safe=safe), self._origin_value)
+
```

Comment:
Can we have a regression test so the CI breaks if anyone removes this code in the future?

Reply:
For sure ! I'll do that.

Reply:
This is done with a new django app view and an iast test to check that all is ok.